drive/tests/query_tests_history.rs

```diff
@@ -128,6 +156,10 @@ pub fn setup(count: u32, seed: u64) -> (Drive, Contract, TempDir) {
 
             let storage_flags = StorageFlags { epoch: 0 };
 
+            // if block_time == 100 && i == 9 {
```

Comment:
Commented code

Reply:
Yes, this still needs to be worked on, and will be next PR (batches still have an issue)

Reply:
Let's add a todo then.